tests/printf_float/main.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2016 Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       print floating point values test application
+ *
+ * This test is supposed to check that floating point values can be
+ * displayed with printf function.
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <assert.h>
+
+int main(void)
+{
+    double floating_point_value = 2016.0317;
+    char result[8] = {0};
+    snprintf(result, sizeof(result), "%.2f", floating_point_value);
+    assert(strcmp(result, "2016.03") == 0);
+
+    printf("Value displayed: %.2f\n", floating_point_value);
```

Comment:
Agreed, I was unsure what was the best thing to display. Looking at other tests, I rewrote this part. With the native build, the `assert` is skipped (-g doesn't seem to be used in most of the cases).